docs/content/comparison-to-other-systems.md

```diff
@@ -430,7 +454,10 @@ roughly the same as for XACML: attributes of users, actions, and resources.
 
 ```live:xacml:module:openable
 package xacml
```

Comment:
If we're looking to modernize this, we can do:

```rego
package xacml

import future.keywords

# METADATA
# title: urn:curtiss:ba:taa:taa-1.1
# description: Policy for Business Authorization category TAA-1.1
default permit := false

permit if {
    # Check that resource has a "NavigationSystem" entry
    input.resource["NavigationSystem"]

    # Check that organization is one of the options
    input.user.organization in ["Packard", "Curtiss"]

    # Check that nationality is one of the options
    input.user.nationality in ["GB", "US"]

    # Check that work_effort is one of the options
    input.user.work_effort in ["DetailedDesign", "Simulation"]
}
```

Reply:
Also, the XACML seems to now use 4 spaces for indentation, which is going to require a lot of horizontal scrolling in the docs. Using 2 spaces is probably a better option.